vico/core/src/main/java/com/patrykandpatrick/vico/core/cartesian/layer/LineFills.kt

```diff
@@ -37,15 +37,24 @@ internal data class SingleLineFill(val fill: Fill) : LineCartesianLayer.LineFill
     verticalAxisPosition: Axis.Position.Vertical?,
   ) {
     with(context) {
-      paint.shader =
+      // ----------------------- WAHOO START --------------------------
+      /*paint.shader =
         fill.shader?.provideShader(
           this,
           layerBounds.left,
           layerBounds.top,
           layerBounds.right,
           layerBounds.bottom,
         )
-      canvas.drawPaint(paint)
+      canvas.drawPaint(paint)*/
+      canvas.drawRect(
```

Comment:
drawRect is about twice as fast as drawPaint and does the exact same thing in this case